test/e2e/storage/pd.go

```diff
@@ -59,7 +59,7 @@ const (
 	minNodes            = 2
 )
 
-var _ = utils.SIGDescribe("Pod Disks", func() {
+var _ = utils.SIGDescribe("Pod Disks [Feature:StorageProviderGCEPD]", func() {
```

Comment:
Despite being named pd.go, I believe this file also tests aws disks. We may need to remove "GCEPD" from the feature name or separate into two files.

Reply:
how about `[Feature:StorageProvider]`? seems like a unique name

Reply:
done